xbmc/filesystem/File.cpp

```diff
@@ -293,23 +293,26 @@ bool CFile::Open(const CURL& file, const unsigned int flags)
              URIUtils::IsInternetStream(pathToUrl, true)) ||
             (iCacheBufferMode == CACHE_BUFFER_MODE_TRUE_INTERNET &&
              URIUtils::IsInternetStream(pathToUrl, false)) ||
-            (iCacheBufferMode == CACHE_BUFFER_MODE_REMOTE && URIUtils::IsRemote(pathToUrl)) ||
-            (iCacheBufferMode == CACHE_BUFFER_MODE_ALL))
+            (iCacheBufferMode == CACHE_BUFFER_MODE_REMOTE &&
```

Comment:
shouldn't this be CACHE_BUFFER_MODE_NETWORK?

Reply:
Yup, thanks for the headsup. Fixed :)